src/backend/distributed/transaction/citus_dist_stat_activity.c

```diff
@@ -273,17 +349,10 @@ Datum
 citus_dist_stat_activity(PG_FUNCTION_ARGS)
 {
 	List *citusDistStatStatements = NIL;
-	StringInfo citusDistStatQuery = NULL;
-	const char *workerQuery = "false";
 
 	CheckCitusVersion(ERROR);
 
-	/* set the workerQuery to false in the query */
-	citusDistStatQuery = makeStringInfo();
-	appendStringInfo(citusDistStatQuery, CITUS_DIST_STAT_ACTIVITY_QUERY,
-					 workerQuery);
-
-	citusDistStatStatements = CitusDistStatActivity(citusDistStatQuery->data);
+	citusDistStatStatements = CitusDistStatActivity(CITUS_DIST_STAT_ACTIVITY_QUERY);
```

Comment:
The function name `CitusDistStatActivity ()` seems confusing. (even though I've implemented that). Shall we rename it to `CitusStatActivity()`?